plugins/org.jkiss.dbeaver.ext.databend/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,19 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.clickhouse;singleton:=true
```

Comment:
All this Manifest file needs to be changed

Reply:
Thanks very much 😄